ampersand-filtered-subcollection.js

```diff
@@ -322,6 +326,11 @@ assign(FilteredCollection.prototype, Events, {
             if (!this._testModel(model) || alreadyHave) {
                 action = 'ignore';
             }
+        } else if (startsWith(action, 'change')) {
```

Comment:
I'd refactor this whole function to use `event`, `eventName` and `propName` instead.
Using `startsWith` will also catch custom events like `changed`, `changedProp` or anything like this.
If not changing whole thing, you can use `eventName.split(':')[0] === 'change'` as it's already used in plenty of places in Ampersand code.